.changes/next-release/bugfix-6871c109-4229-4247-a897-debfba52d980.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Amazon Q Feature Development: Update error message when repo size larger than 200 megabytes"
```

Comment:
issue: I would say this is more a feature as you are adding a check for the repo size prior to uploading the artifacts regardless of using a new error message or not.

Reply:
if other people see it as a feature, i can change it as well. Don't have a strict opinion about it

Reply:
Well, it is a feature right? This check didn't exist in the logic before.

Reply:
I don't see it as a feature, since we were always were failing to treat those repos. But before this CR the user was seeing another error after calling the execute planning API, with no buttons. So this CR is only fixing the Error to be appropriate(with button) + fail fast. 

Let me know if you still see it as a feature, i can change it to feature

Reply:
I see, thanks for the explanation.

Reply:
To me it also looks like a bugfix - previously there was a bug that, repos larger than 200 MB were being sent to BE and we were failing to generate a plan/code for it. Now, it is fixed, and FE doesn't send that repo to BE and fails gracefully.